### ISSUE TITLE
Sugguestion on level advancement logic

### ISSUE THREAD

**Issue by AlienKevin** [web-Nuo/Kobrd#1]
First, I wanna say that this is the best app I encountered so far for learning to type in Korean! The website looks well-polished and the UI looks gorgeous. Thanks for developing this awesome app! One small issue I encountered is that I'm automatically transferred to Game Mode after completing a level. It would be great if I can automatically arrive at the next level. I also wish there are more words to practice for each level.

**Comment by nuooooooooooo** [comment]
Hi Kevin ! Thanks a lot for raising this issue ! I never thought this project would ever be noticed by anyone, so I was really stoked when I noticed I received a message ! (btw I saw you developed an app to practice 漢字, that's seriously so cool !).
Regarding the issue with the levels, I found out it could be fixed easily by switching a while loop for an if statement (rookie mistake ╮(╯▽╰)╭ ). Fixed with this commit : 6cae09a3ec622ef58e9dc19782bce4b7f08837e4
For the other improvement you brought up, regarding level content, I'll look up more material to make the levels more varied so you can get more practise before moving up a step.
Anyway, sorry for rambling a bit, I'm just really happy to receive my first issue ever haha it really made my day !

**Comment by AlienKevin** [comment]
I had this moment too several years ago when someone opened an issue under my repo. Btw, I saw you included Mandarin and Korean on your intro readme. I'm a native Mandarin speaker and I started learning Korean recently out of curiosity (I watched many K dramas, love kimchi, and also heard that ~60% of Korean vocab is imported through Hanzi). I'm interested in how Hanzi is used across languages and varieties so I learned a few months of Cantonese and now started studying Korean. What's your motivation for learning Mandarin & Korean?

**Comment by nuooooooooooo** [comment]
Yeah I've also noticed a lot of similarities between the two ! It definitely helps a lot to have a good level in Mandarin before starting Korean. For Mandarin, at first it was mostly because everyone was saying Mandarin Chinese is the language of the future, I came to like it more and more as I studied, now I can not imagine not having Mandarin in my life haha. For Korean, it's pretty much the same as you, I used to listen to a lot of kpop and I still watch a lot of kdramas, but my speaking skills are still pretty limited because I never really practiced speaking before. I do hope I can get up to TOPIK 5/6 one day. Cantonese sounds really good too, but I guess I'm a bit intimidated by the fact it has even more tones than mandarin 😅

**Comment by AlienKevin** [comment]
Your experience is super cool! I'm also a CS major and am planning to minor in Linguistics. Do you want to continue our conversation on a messaging app? I prefer WhatsApp but I also use Signal and WeChat.

**Comment by nuooooooooooo** [comment]
There doesn't seem to be a way to send a dm on this website, would you mind sending me either your WhatsApp or WeChat ID at my email address manonlocht(at)gmail(dot)com ?

**Comment by AlienKevin** [comment]
Great, I sent my phone number to your email.
